app/jobs/clean_users_job.rb

```diff
@@ -0,0 +1,9 @@
+class CleanUsersJob
+  include SuckerPunch::Job
+
+  def perform(interval)
+    CleanUsersService.call
+
+    CleanUsersJob.perform_in(interval, interval)
```

Comment:
Are there any other alternatives to scheduling recurring jobs like this? My worry is that if one instance of the job blows up, the jobs are going to stop until the next time the process gets rebooted.

A few ideas:
1. Could we use a crontab to schedule this? Something like the `whenever` gem?
2. If not, could we do this next `perform_in` in an `ensure` block so that even if the job blows up, we still schedule the next one?

Reply:
IIRC in planning we talked about making this a button or something similar so we didn't have to worry about tooling for a job.

Reply:
I'm happy to do what you feel is best.  I was admittedly looking for a solution that avoided an external dependency like `crontab`, but I didn't take into account the possibility of the service failure preventing future events.  Thank you very much for pointing this out.  If you're agreeable, I would like to see how well `ensure` addresses your concern, but again, I'm willing to forgo the whole approach if you feel strongly otherwise.

Reply:
My apologies @hooper I didn't see your comment before I wrote mine. It sounds like the button is the agreed approach - I'll update the AC.  Question - shall all dashboard admins be allowed to click this button?  Shall we instead restrict it to a few specific users?